doc/source/cluster/kubernetes/user-guides/kubectl-dashboard.md

```diff
@@ -0,0 +1,33 @@
+(kubectl-dashboard)=
+
+# Use kubectl dashboard (alpha)
+
+Starting from KubeRay v1.4.0, you can use the open source dashboard UI for KubeRay.
+
+## Installation
+
+KubeRay dashboard depends on the optional component `kuberay-apiserver`. So you need to install it first. For simplicity, let's disable the security proxy and allow all origins for CORS.
+
+```bash
+helm install kuberay-apiserver kuberay/kuberay-apiserver --version v1.4.0 --set security= --set cors.allowOrigin='*'
+```
+
+And you need to port-forward the `kuberay-apiserver` service:
+
+```bash
+kubectl port-forward svc/kuberay-apiserver-service 31888:8888
```

Comment:
why do we need to port forward API server?

Reply:
Because currently KubeRay dashboard makes requests to `http://localhost:31888`

Reply:
In that case, is it sufficient for the KubeRay dashboard to connect to the KubeRay API server, or does the browser also need to be able to send requests directly to the KubeRay API server?